build.sh

```diff
@@ -5,11 +5,156 @@
 # This script is basically automation script of dev.md
 # It is supposed to run in same folder where the file lives (root).
 
-# 1. enter into build folder
-mkdir -p build && cd build
+# Unfortunately we still need to manually fix the GRPC issue listed in dev.md
+
+if [ "$(expr substr $(uname -s) 1 5)" != "Linux" ]; then
+  echo "Must be on Linux machine to run this script"
+  exit
+fi
+
+# packages could be installed by apt-get install
+aptGetInstallPackages=(
+  "libcurl4-gnutls-dev"
+  "libunwind-dev"
+  "libiberty-dev"
+  "gnutls-dev"
+  "libgcrypt-dev"
+  "libkrb5-dev"
+  "libldap-dev"
+  "librtmp-dev"
+  "libnghttp2-dev"
+  "libpsl-dev"
+  "libutf8proc-dev"
+  "libgflags-dev"
+  "libboost-dev"
+  "liblz4-dev"
+  "libzstd-dev"
+  "libsnappy-dev"
+  "liblzma-dev"
+  "autoconf"
+  "rapidjson-dev"
```

Comment:
this is not required as we have external dependency defined in `Json_Ext.cmake`

Reply:
I'll remove those 4 then.